internal/topology/variables/clusterclass_variable_validation_test.go

```diff
@@ -2593,6 +2593,109 @@ func Test_ValidateClusterClassVariable(t *testing.T) {
 					"spec.variables[cpu].schema.openAPIV3Schema"),
 			},
 		},
+		{
```

Comment:
```suggestion
		 // oneOf/anyOf/allOf/not
		{
```
nit (we have some comments between blocks of test cases to make it easier to maintain some order)

Reply:
Added.